tg/MainScreenListOfGames/ListOfGamesViewController.swift

```diff
@@ -147,33 +146,24 @@ extension ListOfGamesViewController: UITableViewDelegate, UITableViewDataSource{
 
 extension ListOfGamesViewController: UISearchResultsUpdating {
     func updateSearchResults(for searchController: UISearchController) {
-        filtration(searchController.searchBar.text!)
+        
+        guard let text = searchController.searchBar.text else { return }
```

Comment:
In this case, user won't see updated results. So if `searchController.searchBar.text` is nil you have to set `resultsOfSearch = gameList`